pandas/_libs/tslibs/offsets.pyx

```diff
@@ -3563,6 +3601,9 @@ cpdef to_offset(freq):
     >>> to_offset(Hour())
     <Hour>
     """
+    # TODO: avoid runtime imports
```

Comment:
really really prefer absolute imports

Reply:
sure.  im more concerned about getting rid of the runtime imports